octoprint_v8theme/static/js/v8theme.js

```diff
@@ -1,33 +1,564 @@
+          elementTarget.appendTo("#bed_temp .bed_target");
+          elementOffset.appendTo("#bed_temp .bed_offset");
+        }
+      });
+
+      var tempButton;
+      $(".hotend_target .dropdown-menu li").each(function() {
+        tempButton = $(this).find("a");
+        if (tempButton.text().indexOf('(') !== -1) {
+          tempLabel = tempButton.text().match(/\(([^)]+)\)/)[1];
+          tempButton.text(tempButton.text().replace(tempLabel, "").replace(/ |\(|\)/g, ""));
+        } else {
+          tempLabel = "Off"
+        }
+        tempButton.text(tempButton.text().replace(/Set/g, ""));
+        tempButton.wrapInner("<button class='btn' title='" + tempLabel + "'></button>").appendTo("#hotend_temp .btn-spread");
+      });
+      $(".bed_target .dropdown-menu li").each(function() {
+        tempButton = $(this).find("a");
+        if (tempButton.text().indexOf('(') !== -1) {
+          tempLabel = tempButton.text().match(/\(([^)]+)\)/)[1];
+          tempButton.text(tempButton.text().replace(tempLabel, "").replace(/ |\(|\)/g, ""));
+        } else {
+          tempLabel = "Off"
+        }
+        tempButton.text(tempButton.text().replace(/Set/g, ""));
+        tempButton.wrapInner("<button class='btn' title='" + tempLabel + "'></button>").appendTo("#bed_temp .btn-spread");
+      });
+      $(".hotend_target button.dropdown-toggle, .hotend_target .dropdown-menu").remove();
+      $(".bed_target button.dropdown-toggle, .bed_target .dropdown-menu").remove();
+      $("#temperature_main table.table-bordered").remove();
+
+      $(".btn-spread").tooltip({
+        position: {
+          my: "left top+10",
+          at: "center-15 bottom",
+          collision: "flipfit"
+        }
+      });
+
+      $('.accordion-heading').has('.heading_buttons').click(_.throttle(function (e) {
+        if ($(e.target).is('.heading_buttons') || $(e.target).parents('.heading_buttons').length > 0) { return }
+        if ($(this).next('.accordion-body').hasClass('in')) {
+          $(this).find('.heading_buttons').fadeOut('150');
+        } else {
+          $(this).find('.heading_buttons').fadeIn('150');
+        }
+      }, 375, {trailing: false}, {leading: false}));
+
+      $(".navbar-inner .nav-collapse, .btn-navbar").after("<div class='pull-left-container'><ul class='nav pull-left'><li><img class='printer-icon' src='plugin/v8theme/static/icon-printer.png'/><span class='printer_name_span'></span></li></ul></div>");
+      $(".btn-navbar").wrap("<div class='btn-nav-container'></div>");
+      $.ajax({
+        type: "GET",
+        url: "/api/plugin/v8theme",
+        contentType: "application/json; charset=utf-8",
+        dataType: "json",
+        success: function(response) {
+          if (response.printer_name && response.printer_name != "") {
+            self.setPrinterName(response.printer_name);
+          } else {
+            self.hidePrinterName();
+          }
+        },
+        error: function(jqXHR, exception) {
```

Comment:
I think it needs to be `error: function(jqXHR, textStatus, exception) `
maybe that's why the `404` was getting displayed in the chrome console?
we should try that again